File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/test/PlayerConfig.java
package com.github.kotvertolet.youtubejextractor.models.test;

import com.google.gson.annotations.SerializedName;

public class PlayerConfig{

	@SerializedName("webPlayerConfig")
	private WebPlayerConfig webPlayerConfig;

	@SerializedName("mediaCommonConfig")
	private MediaCommonConfig mediaCommonConfig;

	@SerializedName("daiConfig")
	private DaiConfig daiConfig;

	@SerializedName("audioConfig")
	private AudioConfig audioConfig;

	@SerializedName("streamSelectionConfig")
	private StreamSelectionConfig streamSelectionConfig;

	public void setWebPlayerConfig(WebPlayerConfig webPlayerConfig){
		this.webPlayerConfig = webPlayerConfig;
	}

	public WebPlayerConfig getWebPlayerConfig(){
		return webPlayerConfig;
	}

	public void setMediaCommonConfig(MediaCommonConfig mediaCommonConfig){
		this.mediaCommonConfig = mediaCommonConfig;
	}

	public MediaCommonConfig getMediaCommonConfig(){
		return mediaCommonConfig;
	}

	public void setDaiConfig(DaiConfig daiConfig){
		this.daiConfig = daiConfig;
	}

	public DaiConfig getDaiConfig(){
		return daiConfig;
	}

	public void setAudioConfig(AudioConfig audioConfig){
		this.audioConfig = audioConfig;
	}

	public AudioConfig getAudioConfig(){
		return audioConfig;
	}

	public void setStreamSelectionConfig(StreamSelectionConfig streamSelectionConfig){
		this.streamSelectionConfig = streamSelectionConfig;
	}

	public StreamSelectionConfig getStreamSelectionConfig(){
		return streamSelectionConfig;
	}
}

File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/newModels/PlayerConfig.kt
package com.github.kotvertolet.youtubejextractor.models.newModels

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize
import java.io.Serializable

@Parcelize
class PlayerConfig : Parcelable, Serializable {
    @SerializedName("webPlayerConfig")
    var webPlayerConfig: WebPlayerConfig? = null

    @SerializedName("mediaCommonConfig")
    var mediaCommonConfig: MediaCommonConfig? = null

    @SerializedName("daiConfig")
    var daiConfig: DaiConfig? = null

    @SerializedName("audioConfig")
    var audioConfig: AudioConfig? = null

    @SerializedName("streamSelectionConfig")
    var streamSelectionConfig: StreamSelectionConfig? = null
    override fun toString(): String {
        return "PlayerConfig{" +
                "webPlayerConfig = '" + webPlayerConfig + '\'' +
                ",mediaCommonConfig = '" + mediaCommonConfig + '\'' +
                ",daiConfig = '" + daiConfig + '\'' +
                ",audioConfig = '" + audioConfig + '\'' +
                ",streamSelectionConfig = '" + streamSelectionConfig + '\'' +
                "}"
    }
}